src/BinSkim.Sdk/CompilerDataLogger.cs

```diff
@@ -27,6 +28,8 @@ public class CompilerDataLogger
         private static TelemetryClient s_telemetryClient;
         private static TelemetryConfiguration s_telemetryConfiguration;
 
+        public static bool Enabled => s_telemetryClient != null;
```

Comment:
>Enabled [](http://example.com/codeflow?start=27&length=7)

Let's rename for TelemetryEnabled?
AppInsightsEnabled?

Reply:
something more specific.

Reply:
like TelemetryEnabled